_config.yml

```diff
@@ -99,3 +99,18 @@ comments:
     ClientSecret:
     lazy: true
 
+  gitalk:
```

Comment:
![image](https://user-images.githubusercontent.com/4197468/53678888-1ef24900-3d00-11e9-9847-94a92949419d.png)
Be careful with the spaces in front of the variable start define

Reply:
I'm not sure that I understand your meanings.
In front of 'gitalk:', there are **two space** because the [gitalk] is inside [comments] option.
And you can check the code in `comment.ejs`  :
```js
<% } else if (theme.comments.gitalk.clientID && theme.comments.gitalk.clientSecret ) { %>
```

Feel free when you share your opinion :)